data/tutorials/gs_00_up_and_running.md

```diff
@@ -211,57 +257,66 @@ While a toplevel is great for interactively trying out the language, we will sho
 
 OCaml has plugins for many editors, but the most actively maintained are for Visual Studio Code, Emacs, and Vim.
 
-For **Visual Studio Code**, we install the [OCaml Platform Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ocamllabs.ocaml-platform) from the Visual
+For **Visual Studio Code**, install the [OCaml Platform Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ocamllabs.ocaml-platform) from the Visual
 Studio Marketplace.
 
 Upon first loading an OCaml source file, you may be prompted to select the
-toolchain in use: pick the version of OCaml you are using, e.g., 4.11.1
+toolchain in use: pick the version of OCaml you are using, e.g., 5.0.0
 from the list. Now, help is available by hovering over symbols in your program:
 
 ![Visual Studio Code](/media/tutorials/vscode.png)
 
-**For Vim and Emacs** we won't use the LSP server, but rather directly talk to Merlin, which we installed earlier.
+**For Vim and Emacs** we won't use the LSP server but rather directly talk to Merlin.
 
-When installing Merlin, instructions were printed on how to link Merlin with your editor. If you do not have them visible, the short way is just to run:
+When we installed Merlin above, instructions were printed on how to link Merlin with your editor. If you do not have them visible, the short way is just to run:
 
 ```shell
-opam user-setup install
+$ opam user-setup install
 ```
 
 ## Starting a New Project
 
-We explain here all that's needed to start a project using the tools that we installed. The build system, Dune, allows us to initialise a project, containing a `helloworld` example:
+Let's begin the simplest project with Dune and OCaml, "Hello World," just to see how it works. 
+Navigate into a practice directory, as we'll be creating new projects below. 
+
+First, we ask `dune` to
+initialize a new project, and then change into the created directory. As usual, 
+the line beneath the command is the desired output:
 
 ```shell
-dune init project helloworld
-cd helloworld
+$ dune init project helloworld
+Success: initialized project component named helloworld
+$ cd helloworld
 ```
 
 All the metadata of your project is available in the file `dune-project`. Edit it to match your specific project.
 
-We can build our program with `dune build`:
+We can build our program with `dune build`, which creates an executable file:
 
 ```shell
-dune build
+$ dune build
 ```
 
 When we change our program, we can type `dune build` again to make a new
 executable. To run the program, we can use:
 
 ```shell
-dune exec ./bin/main.exe
+$ dune exec /bin/main.exe
```

Comment:
For me, it worked with the `./` (and it should work!). However, `dune exec /bin/main.exe` definitely should not work and would output `Error: Program "/bin/main.exe" not found!`!

Let's remove the useless `./`:
```suggestion
$ dune exec bin/main.exe
```